backend/functions/lib/controllers/case-history/case-history.controller.test.ts

```diff
@@ -8,42 +8,55 @@ import {
   THROW_UNKNOWN_ERROR_CASE_ID,
 } from '../../testing/testing-constants';
 import { NotFoundError } from '../../common-errors/not-found-error';
+import { CamsHttpRequest } from '../../adapters/types/http';
+import { mockCamsHttpRequest } from '../../testing/mock-data/cams-http-request-helper';
 
 describe('Test case-history controller', () => {
   let applicationContext;
 
   beforeEach(async () => {
-    applicationContext = await createMockApplicationContext({ DATABASE_MOCK: 'true' });
+    applicationContext = await createMockApplicationContext();
   });
 
   test('should return a case history when getCaseHistory is called', async () => {
     jest.spyOn(CaseHistoryUseCase.prototype, 'getCaseHistory').mockResolvedValue(CASE_HISTORY);
     const caseId = NORMAL_CASE_ID;
+    const request: CamsHttpRequest = mockCamsHttpRequest({
+      params: { caseId },
+    });
+    applicationContext.request = request;
     const controller = new CaseHistoryController(applicationContext);
-    const result = await controller.getCaseHistory(applicationContext, { caseId });
-    expect(result.isSuccess).toBeTruthy();
-    expect(result['data']).toEqual(CASE_HISTORY);
+    const result = await controller.getCaseHistory(applicationContext, request);
```

Comment:
We can probably update this, since request is probably being passed in within the applicationContext, the controller or use case should grab it off the applicationContext.